bigtable_input.py

```diff
@@ -0,0 +1,135 @@
+"""Read Minigo game examples from a Bigtable.
+"""
+
+import logging
+import os
+import struct
+from google.cloud import bigtable
+from google.cloud.bigtable import row_filters
+import tensorflow as tf
+
+
+_project_name = os.environ['PROJECT']
+_instance_name = os.environ['CBT_INSTANCE']
+_table_name = os.environ['CBT_TABLE']
+
+_bt_table = bigtable.Client(
+    _project_name).instance(
+        _instance_name).table(
+            _table_name)
+
+_tf_table = tf.contrib.cloud.BigtableClient(
+    _project_name,
+    _instance_name).table(
+        _table_name)
+
+
+def get_latest_game_number():
+    """Return the number of the next game to be stored.
+
+    The state of the table is stored in `metadata:game_counter`
+    in the `table_state` row only.
+    """
+    counter = b'game_counter'
```

Comment:
can this be inlined as a string like 'table_state' and 'metadata'?

Reply:
The reason I didn't is because it's referenced 3 times in the code, and does need to be specified as a byte string (unlike the column family and row name, oddly). I don't have a problem with expanding it in this case, since the scope is so small.